microsoft-edge/devtools-guide-chromium/progressive-web-apps/protocol-handlers.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Test Progressive Web App (PWA) protocol handling
+description: Use the Application tool to test custom protocols you've defined in your PWA web app manifest.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 08/23/2022
+---
+
+# Test Progressive Web App (PWA) protocol handling
+
+This article assumes that you have already defined protocol handlers in your PWA web app manifest and are debugging your app with the DevTools. To learn about how to define and register protocols in your PWA see [Handle protocols in Progressive Web Apps](../../progressive-web-apps-chromium/how-to/handle-protocols.md).  
```

Comment:
@zoherghadyali just as an FYI, I removed the note formatting here. We tend to avoid using them too much, as we've learned that people actually skip over them.